canonical/lxd-ui
# Retry for network changes on localhost

## Done

- requests to alter networks on localhost will fail, add retry logic for those cases. The problem only occurs when talking directly to lxd (without dev server) on localhost via https://localhost:8443/ui/

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check network editing/renaming/creation/deletion
    - the error can only be triggered when talking directly to LXD on localhost (without the dev server), maybe not worth to find a way to set this up, we can verify the fix from the nightly LXD build after it is merged.
